xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -390,7 +390,7 @@ For more information, see [Exclude all null value properties](/dotnet/standard/s
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Determines whether read-only fields are ignored during serialization. A property is read-only if it isn't marked with the <see langword="readonly" /> keyword. The default value is <see langword="false" />.</summary>
+        <summary>Determines whether read-only fields are ignored during serialization. A field is read-only if it isn't marked with the <see langword="readonly" /> keyword. The default value is <see langword="false" />.</summary>
```

Comment:
"if it isn't" -- should this read "if it is"?

Reply:
Yup, thanks.